src/drivers/io/vesa_log.zig

```diff
@@ -164,9 +194,15 @@ fn feed_line() void {
   }
 }
 
+pub fn update() void {
+  var y = @truncate(u32, framebuffer.?.pos_y * font.height);
+  framebuffer.?.updater(0, y, framebuffer.?.width, @truncate(u32, font.height), framebuffer.?.updater_ctx);
+}
+
 pub fn putch(ch: u8) void {
   if(framebuffer != null) {
     if(ch == '\n') {
+      update();
```

Comment:
Why do we need to call `update()` here? The only way this changes the contents on the screen is if `feed_line()` calls `scroll_fb()`, which would call the updater anyways.